BUILD.md

```diff
@@ -75,10 +75,10 @@ Build the latest container:
 python3 ./install/common/build-image.py
 ```
 
-Download the OCR language data:
+Download the necessary assets:
```

Comment:
```suggestion
Dangerzone depends on some assets that should be downloaded in order to run (think binaries and others resources). This can be done with the following command:
```